addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComRfyMessage.java

```diff
@@ -29,17 +31,19 @@
  *
  * @author Jürgen Richtsfeld - Initial contribution
  * @author Pauli Anttila
+ * @author Mike Jagdis
```

Comment:
Maybe add that pauli ported it to OH2 (maybe check with a `blame`)
And add a description of your own contribution